cinch/library/jenkins_plugin.py

```diff
@@ -0,0 +1,826 @@
+#!/usr/bin/python
```

Comment:
Noting here: Ansible 2.2.1 includes a version of jenkins_plugin.py that addresses this bug, along with a few other minor issues. I am going to create a separate issue to drop this file once we upgrade to a version of linchpin that depends on Ansible 2.2.1, and can be guaranteed of the fixed version of Ansible being in our path.